### PR TITLE
Docs about values

### PR DESCRIPTION
**Pull request by kulmann**

### CODE REVIEW

File: docs/values.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+provided by this service.
+
+## Identifying settings values
+
+A *settings value* is uniquely identified by four attributes. Three of them are coming from the definition of
+the setting within it's settings bundle (see [Settings Bundles](https://owncloud.github.io/extensions/ocis_settings/bundles/)
+for an example). The fourth identifies the user.
+- extension: Key of the extension that registered the settings bundle,
+- bundleKey: Key of the settings bundle,
+- settingKey: Key of the setting as defined within the bundle,
```
Comment (refs) [inline docs/values.md:21]: seeing it in the docs, would it make sense to either add / remove the `Key` suffix to all of them for consistency?

Comment (refs) [reply docs/values.md:21]: I'm an advocate for removing it. I'm aware of all the bits and bobs necessary for removing it 😬

File: docs/glossary.md [hunk 2]
```diff
@@ -33,3 +33,10 @@ In the context of this extension and oCIS in general, we are using the following
 
 - Collection of related settings
 - Registered by an ocis extension
+
+### Settings Value
+
+- Manifestation of a setting for a specific user
+- E.g. used for customization (at runtime) in `ocis-web`
+- `ocis-web` extension for modifying settings values is provided by this service
```
Comment (refs) [inline docs/glossary.md:41]: oh, I wasn't aware of this one. is `ocis-web` an extension on the frontend? or is it actually an ocis extension?

note: we should make the term extension less ambiguous, I identify 2 things:

1. ocis extension (i.e: `ocis-konnectd`, `ocis-proxy`...)
2. phoenix extension _point_?

Comment (kulmann) [reply docs/glossary.md:41]: `ocis-web` = phoenix
`ocis-web-settings` = ocis-web extension or app. but we agreed to call it extension.

the term `extension points` is already used in ocis-web for e.g. the indicators extension points = possibility to inject additional components into a web ui.

What I meant here in the glossary, is, that ocis-settings delivers the settings.js as ocis-web extension, i.e. `ocis-web-settings`.

File: docs/values.md (hunk 1 above)
Comment (kulmann) [reply docs/values.md:21]: Should we discuss this in the sprint planning? I could agree with removing `key`, but that needs PRs in phoenix, owncloud-sdk, ocis-accounts and ocis-hello as well. Definitely better to do this now, if we want to change it.

Comment (kulmann) [reply docs/values.md:21]: 👍 for consistency

File: docs/glossary.md (hunk 2 above)
Comment (kulmann) [reply docs/glossary.md]: Fixed the extension name

File: docs/values.md (hunk 1 above)
Comment (kulmann) [reply docs/values.md:21]: Raised ticket here: https://github.com/owncloud/ocis-settings/issues/32

File: docs/glossary.md (hunk 2 above)
Comment (refs) [reply docs/glossary.md]: updating my personal knowledge base with `ocis-web === phoenix`. Thank you!

**Pull request merged**